### DISCUSSION TITLE
License Warning

### DISCUSSION THREAD

**Discussion by BryanDollery** [yarnpkg/berry#5087] [Q&A] [answered]
I'm using Yarn in a private project to pull down my nodejs application dependencies because it's faster than npm. However, when I do this I get a warning:

```
warning package.json: License should be a valid SPDX license expression
```

Because I have the following line in my `package.json` file:

```json
"license": "PRIVATE",
```

I can't find a way to suppress this warning. 

Is this a bug? Is there a way to stop the warning?

**Comment by devoto13** [comment, marked as answer]
Maybe `UNLICENSED` is the way? See https://docs.npmjs.com/cli/v8/configuring-npm/package-json#license.

**Reply by BryanDollery** [reply]
Yup, that fixed it. Thanks for the help.
